lxd/container_lxc.go

```diff
@@ -1130,12 +1130,17 @@ func (c *containerLXC) startCommon() (string, error) {
 					}
 				}
 
-				m["major"] = fmt.Sprintf("%d", usb.major)
-				m["minor"] = fmt.Sprintf("%d", usb.minor)
-				m["path"] = usb.path
+				temp := shared.Device{}
+				if err := shared.DeepCopy(&m, &temp); err != nil {
+					return "", err
+				}
+
+				temp["major"] = fmt.Sprintf("%d", usb.major)
+				temp["minor"] = fmt.Sprintf("%d", usb.minor)
+				temp["path"] = usb.path
```

Comment:
I may be missing something, but this looks identical to createUSBDevice. Why not use it?